full_name.rb

```diff
@@ -0,0 +1,15 @@
+def Ask(question)
+    puts question
+    string = gets.chomp + " "
```

Comment:
rather than adding a space to the end of the word here I would probably wait till the end of the method and just use `join(' ')`